frontend/src/components/workbench/panel/workflow-panel.tsx

```diff
@@ -276,10 +276,6 @@ export function WorkflowPanel({
                                 />
                               )}
                             </div>
-                            <FormDescription className="text-xs">
-                              A name that can be used to uniquely identify this
-                              workflow.
-                            </FormDescription>
```

Comment:
could we maybe have a tooltip that tells you what aliases are used for?

Reply:
Can do